index.html

```diff
@@ -54,6 +54,197 @@ <h1 class="head">I’m Mariane<br>Glad to see<br> you!</h1>
+                </li>
+                <li>
+                    <h2 class="year">2015</h2>
+                </li>
+              </ul>
+            </div>
+            <p class="p-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang">
+                <a class="p-html" href="">html</a>
+                <a class="p-css" href="">css</a>
+                <a class="p-javas" href="">javaScript</a>
+            </div>
+            <div class="p-button">
+                <a class="p-proj" href="">See Project</a>
+            </div>
+           
+        </section>
+        <section class="card2">
+            <section class="portfolio-image4"></section>
+            <div class="p-layout">
+              <h1 class="portfolio-head">Tonic</h1>
+              <ul class="frame">
+                  <li>
+                      <h2 class="canopy">CANOPY</h2>
+                  </li>
+                  <li>
+                      <img src="Counter.png" alt="">
+                  </li>
+                  <li>
+                    <h2 class="backend">Back End Dev</h2>
+                  </li>
+                  <li>
+                    <img src="Counter.png" alt="">
+                </li>
+                <li>
+                    <h2 class="year">2015</h2>
+                </li>
+              </ul>
+            </div>
+            <p class="p-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang">
+                <a class="p-html" href="">html</a>
+                <a class="p-css" href="">css</a>
+                <a class="p-javas" href="">javaScript</a>
+            </div>
+            <div class="p-button">
+                <a class="p-proj" href="">See Project</a>
+            </div>
+           
+        </section>
+    </div>
+    <div class="grid-container2">
+        <section class="card3">
+            <h1 class="a-head">About<br>Myself</h1>
+            <p class="a-par">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="supporttext">
+                <b class="suptext2">
+                    LET'S CONNECT
+                </b>
+            </div>
+            <ul class="social2"> 
+                <li >
+                    <img src="Vector.svg" alt="">
+                </li>
+                <li>
+                    <img src="Linkedinicon.svg" alt="">
+                </li>
+                <li>
+                    <img src="Vector.png" alt="">
+                </li>
+                <li>
+                    <img src="Vector2.svg" alt="">
+                </li>
+                <li>
+                    <img src="Vector3.svg" alt="">
+                </li>
+            </ul>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Kindly adjust this part.